pkg/asset/installconfig/aws/session.go

```diff
@@ -198,12 +202,14 @@ func getUserCredentials() error {
 
 	creds, err := ini.Load(path)
 	if err != nil {
-		if os.IsNotExist(err) {
-			creds = ini.Empty()
-			creds.Section("").Comment = "https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html"
-		} else {
-			return err
+		if !os.IsNotExist(err) {
+			if err2 := os.Remove(path); err2 != nil {
```

Comment:
If IIUC, this will (attempt to) remove a file at `AWS_SHARED_CREDENTIALS_FILE` if it is not a valid ini file. I don't think we should be deleting user's data. It would be safer to bomb out and return a good error message so the user can correct the issue.